library/src/androidTest/java/org/xmtp/android/library/InstrumentedTest.kt

```diff
@@ -123,91 +97,4 @@ class InstrumentedTest {
-            InvitationV1.newBuilder().build().createRandom(context = invitationContext)
-        val senderBundle = client.privateKeyBundleV1?.toV2()
-        assertEquals(
-            senderBundle?.identityKey?.publicKey?.recoverWalletSignerPublicKey()?.walletAddress,
-            fakeWallet.address
-        )
-        val invitation = SealedInvitationBuilder.buildFromV1(
-            sender = client.privateKeyBundleV1!!.toV2(),
-            recipient = contact.toSignedPublicKeyBundle(),
-            created = created,
-            invitation = invitationv1
-        )
-        val inviteHeader = invitation.v1.header
-        assertEquals(inviteHeader.sender.walletAddress, fakeWallet.address)
-        assertEquals(inviteHeader.recipient.walletAddress, fakeContactWallet.address)
-        val header = SealedInvitationHeaderV1.parseFrom(invitation.v1.headerBytes)
-        val conversation =
-            ConversationV2.create(client = client, invitation = invitationv1, header = header)
-        assertEquals(fakeContactWallet.address, conversation.peerAddress)
-
-        conversation.send(content = "hello world")
-
-        val conversationList = client.conversations.list()
-        val recipientConversation = conversationList.lastOrNull()
-
-        val messages = recipientConversation?.messages()
-        val message = messages?.firstOrNull()
-        if (message != null) {
-            assertEquals("hello world", message.body)
-        }
-    }
-
-    @Test
-    @Ignore
-    fun testCanReceiveV1MessagesFromJS() {
-        val wallet = TestHelpers.FakeWallet.generate()
-        val options =
-            ClientOptions(api = ClientOptions.Api(env = XMTPEnvironment.LOCAL, isSecure = false))
-        val client = Client().create(account = wallet, options = options)
-        val convo = ConversationV1(
-            client = client,
-            peerAddress = "0xf4BF19Ed562651837bc11ff975472ABd239D35B5",
-            sentAt = Date()
-        )
-        convo.send(text = "hello from kotlin")
-        Thread.sleep(1_000)
-        val messages = convo.messages()
-        assertEquals(1, messages.size)
-        assertEquals("hello from kotlin", messages[0].body)
-    }
-
-    @Test
-    @Ignore
-    fun testCanReceiveV2MessagesFromJS() {
-        val wallet = PrivateKeyBuilder()
-        val options =
-            ClientOptions(api = ClientOptions.Api(env = XMTPEnvironment.LOCAL, isSecure = false))
-        val client = Client().create(account = wallet, options = options)
-        client.publishUserContact()
-        val convo = client.conversations.newConversation(
-            "0xf4BF19Ed562651837bc11ff975472ABd239D35B5",
-            InvitationV1ContextBuilder.buildFromConversation("https://example.com/4")
-        )
-
-        convo.send(content = "hello from kotlin")
-        Thread.sleep(1_000)
-        val messages = convo.messages()
-        assertEquals(1, messages.size)
-        assertEquals("hello from kotlin", messages[0].body)
-    }
```

Comment:
I moved these into unit tests since they didn't need to run on a blocking thread.